src/structures/MessageSelectMenu.js

```diff
@@ -168,8 +168,8 @@ class MessageSelectMenu extends BaseMessageComponent {
   }
 
   /**
-   * Transforms this select menu to a plain object
-   * @returns {Object} The raw data of this select menu
+   * Transforms the select menu to a plain object
```

Comment:
I think "this" sounds better here

Reply:
This is inconsistent with every other description

Reply:
well then it should say into instead cuz this sounds a bit weird imo
```suggestion
   * Transforms the select menu into a plain object
```